src/gamestate/dcon_generated.txt

```diff
@@ -4607,6 +4617,11 @@ object {
 	size{ 300000 }
 	tag{ save }
 
+	property{
```

Comment:
this should be removed as it isn't part of the PR, whatever you intend for it

Reply:
Fixed w #13ae911

Reply:
This is still in your PR / didn't get removed in your latest commit